### PR TITLE
Revert "Update precompile address"

### PR DESCRIPTION
**Pull request by wuminzhe**
Reverts darwinia-network/crab-docs#35

### CODE REVIEW

**Comment by HackFisher** [comment]
It's OK, we can continue with #35 in this PR, the request remains valid for @AsceticBear

**Comment by HackFisher** [comment]
The address changes are needed, don't have to revert, but we need more to tell contract developers about how to use the precompile contract.

**Comment by AsceticBear** [comment]
>  we need more to tell contract developers about how to use the precompile contract.

let me add more details later in this pull request.

File: docs/dvm-system-contract.md [hunk 1]
```diff
@@ -28,4 +27,83 @@ The system contracts are deployed in various networks as follows.
...
 | Identity      | 0x0000000000000000000000000000000000000004 |
-| WithDraw      | 0x0000000000000000000000000000000000000015 |
\ No newline at end of file
+| WithDraw      | 0x0000000000000000000000000000000000000015 |
+
+> The Transfer contract is a superset of the Withdraw contract, and the Crab network's Withdraw contract will be upgraded to the Transfer contract in the future.
+
+## Contract Docs
+
+In order to be as compatible as possible with Ethereum and reduce the burden for application migration, the system contract currently consists of two types, the pre-compiled contracts compatible with Ethereum(ECRecover, Sha256, Ripemd160, Identity) and the Pangolin-specific pre-compiled contract Transfer.
```
Comment (wuminzhe) [inline docs/dvm-system-contract.md:36]: Pangolin-specific or Darwinia-specific?

File: docs/dvm-system-contract.md [hunk 2]
```diff
@@ -28,4 +27,83 @@ The system contracts are deployed in various networks as follows.
...
+
+### Transfer Contract
+
+Transfer is a Pangolin-specific pre-compiled contract with address `0x0000000000000000000000000000000000000015` (reserved for other Ethereum contracts) that is used to transfer value between DVM and Substrate account. The current Transfer contract has two main features:
+
+1. support DVM PRING asset transfer to Substrate account. 
+2. support KTON asset transfer between DVM and Substrate account.
```
Comment (wuminzhe) [inline docs/dvm-system-contract.md:43]: 1. Transfer PRING from DVM to Substrate account.
2. Transfer PKTON between DVM and Substrate account.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/dvm-system-contract.md [hunk 3]
```diff
@@ -27,4 +26,86 @@ sidebar_label: 系统合约
...
+## 合约说明
+
+为了尽可能地兼容以太坊，为应用迁移减少负担，系统合约目前共包括两种类型，分别是兼容以太坊的预编译合约（ECRecover，Sha256，Ripemd160，Identity） 和 Pangolin 特有的预编译合约 Transfer。
+
+### Transfer 合约
+
+Transfer 则是 Pangolin 特有的预编译合约，合约地址 `0x0000000000000000000000000000000000000015`（预留地址给以太坊其他合约），用于 在 DVM 账户和 Substrate 账户之间进行价值转移。目前 Transfer 合约的主要功能有两个：
+
+1. 支持 DVM PRING 资产转移到 Substrate 账户。
+2. 支持 KTON 资产在 DVM 和 Substrate 账户之间转移。
```
Comment (wuminzhe) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/dvm-system-contract.md:42]: PKTON?

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/dvm-system-contract.md [hunk 4]
```diff
@@ -27,4 +26,86 @@ sidebar_label: 系统合约
...
+1. `from`: 需要转移 PRING 资产的 DVM 账户
+2. `to`: Transfer 合约地址，切记不可填错，否则资产无法找回
+3. `data`: Substrate 账户地址
+4. `value`: 资产金额
+
+* PKTON 资产转移
+
+
+同样，用户通过发交易到 Transfer 合约实现 KTON 资产在 DVM 账户和 Substrate 账户之间转移。与 PRING 资产转移不同的是，
+在进行资产转移之前，必须要部署一个 [WKTON](https://github.com/evolutionlandorg/token-contracts/blob/dev/src/WCKTON.sol) 合约，并记录该合约地址（很重要，稍后会用到）。
```
Comment (wuminzhe) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/dvm-system-contract.md:80]: > 在进行资产转移之前，必须要部署一个 [WKTON]
这个能加了些解释么？

File: docs/dvm-system-contract.md (hunk 1 above)
Comment (AsceticBear) [reply docs/dvm-system-contract.md:36]: This doc mainly explains Pangolin or Crab network design. The Darwinia network has its own doc site. So, I think `Pangolin-specific` would be better here.

**Pull request merged**